pattern-1/artifacts/deploy-openshift.sh

```diff
@@ -55,6 +58,7 @@ echo 'deploying databases ...'
 oc create -f rdbms/rdbms-persistent-volume-claim.yaml
 oc create -f rdbms/rdbms-service.yaml
 oc create -f rdbms/rdbms-deployment.yaml
+sleep 10s
```

Comment:
This sleep time is not needed. Before we are deploying deployments, we have used some sleep time. Did u encounter any issues?